apps/lib/apps.c

```diff
@@ -691,8 +691,9 @@ int load_cert_certs(const char *uri,
     if (ret) {
         if (pcert != NULL)
             warn_cert(uri, *pcert, 0, vpm);
-        warn_certs(uri, *pcerts, 1, vpm);
-    } else {
+        if (pcerts != NULL)
+            warn_certs(uri, *pcerts, 1, vpm);
+    } else if (pcerts != NULL) {
         OSSL_STACK_OF_X509_free(*pcerts);
```

Comment:
This also fails for the same reason.  However, see #17488 :)

Reply:
Ah, I was not aware that you were already handling the same issue(s).
My PR is the same, just more compact, using `else if`.

Reply:
Oops, you're right you did handle this case.

Reply:
I don't mind which is used.